lib/view/pages/home_screen.dart

```diff
@@ -0,0 +1,139 @@
+                    Image.asset('assets/images/marker.png'),
+                    const SizedBox(width: 5),
+                    MyText(
+                      data: 'Mansoura, Egypt',
+                      color: kDarkGrayColor,
+                      fontWeight: FontWeight.bold,
+                      fontSize: 15,
+                    ),
+                  ],
+                ),
+              ),
+              const SizedBox(height: 10),
+              const SearchField(),
+              const SizedBox(height: 20),
+              // Image Slider
+              Container(
+                margin: const EdgeInsets.only(bottom: 20),
+                child: CarouselSlider(
+                  options: CarouselOptions(
+                    height: 200,
+                    viewportFraction: 1.0,
+                    autoPlay: true,
+                    autoPlayInterval: const Duration(seconds: 3),
+                    autoPlayAnimationDuration: const Duration(milliseconds: 800),
+                    autoPlayCurve: Curves.fastOutSlowIn,
+                    pauseAutoPlayOnTouch: true,
+                  ),
+                  items: sliderImages.map((imagePath) {
+                    return Builder(
+                      builder: (BuildContext context) {
+                        return Image.asset(
+                          imagePath,
+                          fit: BoxFit.cover,
+                        );
+                      },
+                    );
+                  }).toList(),
+                ),
+              ),
+
+              SectionHeader(
+                title: 'Popular Cars',
+                onPressed: () {},
+              ),
+              SizedBox(
+                height: 250,
+                child: ListView.builder(
+                  itemCount: popularCars.length,
+                  scrollDirection: Axis.horizontal,
+                  itemBuilder: (context, index) => ProductCard(
+                    title: popularCars[index].title,
+                    description: popularCars[index].description,
+                    price: popularCars[index].price,
+                    image: popularCars[index].image,
+                  ),
+                ),
+              ),
+
+              SectionHeader(
+                title: 'Best Selling',
+                onPressed: () {},
+              ),
+              SizedBox(
+                height: 250,
+                child: ListView.builder(
+                  itemCount: bestSelling.length,
+                  scrollDirection: Axis.horizontal,
+                  itemBuilder: (context, index) => ProductCard(
+                    title: bestSelling[index].title,
+                    description: bestSelling[index].description,
+                    price: bestSelling[index].price,
+                    image: bestSelling[index].image,
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
```

Comment:
you can create file prodcut_model.dart and put it in data/model directory

Reply:
Done